### PR TITLE
Quote classPath to deal with spaces in workspaceRootPath

### LINKED ISSUE #139: Error executing PMD when project folder uses spaces
**Describe the bug**
The bug is causing issues where it tells you the following error.
image

**Apex PMD Output**
My settings are:
image
image
image

**Apex PMD Settings**
I have applied to all workspaces/and user profile using the extension

- pmdPath: PMD 7.0.0-rc3 (I am in the process of clearing and removing to see if it will change - all worked fine yesterday) 
- rulesetPath: 

**Apex File**

All files it happens

**Screenshots**
see above
**Versions (please complete the following information):**
 - OS: Windows
 - vscode: 1.81.1

### PR DESCRIPTION
**Pull request by adangel**
Fixes #139

### CODE REVIEW

File: src/lib/apexPmd.ts [hunk 1]
```diff
@@ -140,18 +141,21 @@ export class ApexPmd {
       ...additionalClassPaths,
     ].join(CLASSPATH_DELM);
 
-    let env : NodeJS.ProcessEnv = {...process.env};
-    env["CLASSPATH"] = classPath;
+    let env : NodeJS.ProcessEnv = {};
+    env["CLASSPATH"] = `"${classPath}"`;
```
Comment (adangel) [inline src/lib/apexPmd.ts:145]: That's the actual fix.
I've changed the log output as well to see more info in case there are further problems.

**Comment by ChuckJonas** [comment]
Looks good to me.   After you merge everything in milestone `6.1`, ping me and I can pull & publish.

**Comment by adangel** [comment]
> Looks good to me. After you merge everything in milestone `6.1`, ping me and I can pull & publish.

@ChuckJonas I've made the release now on github using the tag `v0.6.1`. You can either rebuilt it from the tag or just use the release artifact from https://github.com/ChuckJonas/vscode-apex-pmd/releases/tag/v0.6.1 (I've downloaded the vsix from the actions workflow and reuploaded it to the release).

**Comment by adangel** [comment]
@ChuckJonas If you want, you can also add me to your publisher account, then I can publish the new extension myself. My user id is: `0d07caf5-3ed2-6cac-a224-0834485c7cc9`.

**Comment by ChuckJonas** [comment]
@adangel done.

let'er rip!

**Comment by adangel** [comment]
@ChuckJonas Thanks! It worked - 0.6.1 is now available via the marketplace :rocket:

**Pull request merged**